### ISSUE TITLE
Red Hat Contribution of IaC of OpenShift for GCP

### ISSUE THREAD

**Issue by mcleo-d** [finos/compliant-financial-infrastructure#201]
- Noting #205 is related to this issue.

**Comment by AdrianHammond** [comment]
Thanks @mcleo-d for opening this issue. I will take ownership of this from Red Hat.

**Comment by mcleo-d** [comment]
Thanks @AdrianHammond - It would be awesome if you could attend #203 to represent the work on the call and how we can align to the CDMC.

**Comment by AdrianHammond** [comment]
I will be on Fridays call @mcleo-d

**Comment by AdrianHammond** [comment]
Forked dev branch to https://github.com/AdrianHammond/compliant-financial-infrastructure

**Comment by AdrianHammond** [comment]
[Draft pull request](https://github.com/finos/compliant-financial-infrastructure/pull/205) opened to update OpenShift service accelerators to reflect latest release of OpenShift (4.10).

**Comment by AdrianHammond** [comment]
Update 21/04: Work continues on documentation and code to implement service accelerator policies on OCP on GCP. We are working with [compliance as code](https://github.com/ComplianceAsCode) community to create a FINOS policy profile which can be used to scan/report and remediate using the [OpenShift Compliance Operator](https://docs.openshift.com/container-platform/4.10/security/compliance_operator/compliance-operator-supported-profiles.html).

**Comment by AdrianHammond** [comment]
![OCP on GCP Install_FINOS](https://user-images.githubusercontent.com/12713247/164725132-3f5b94de-1443-4205-b005-ccc6d5b56d56.jpeg)

**Comment by AdrianHammond** [comment]
![OCP on GCP Install_FINOS](https://user-images.githubusercontent.com/12713247/164728238-374cca45-b0ae-40ff-9fd7-ab08b59fc28e.jpeg)

**Comment by AdrianHammond** [comment]
17th June : under review targeting raising PR in early July.

**Comment by AdrianHammond** [comment]
@eddie-knight @abdullahgarcia @peter-thomas-db  - initial PR #235 for OCP on GCP. Happy to discuss, I will be joining Friday project call. @mcleo-d

**Comment by AdrianHammond** [comment]
@mcleo-d  Ahead of the project meeting today I thought I would document my thoughts on next steps to complete this issue, below assumes that #235 is merged. 

1. Update  OpenShift SAA (#229) and make any required updates
2. Update documentation post release of OCP 4.11 expected August
3. Review / implement IAC and automation for OCP on GCP
4. Work with @peter-thomas-db and DB team on their deployment of OCP on GCP and review / adopt any additional compliance / policies they have adopted beyond CIS benchmark.

In parallel to item 3 and 4 we will also look at OCP on other cloud platforms - AWS, Azure, IBM Cloud, other?

**Comment by AdrianHammond** [comment]
Hi @mcleo-d 

I think it might be possible to close this off as with the work already done with merge #235 and the following new issues

1. #253 to create IAC for OCP on GCP
2. #229 Updating OCP SAA to new template

I think the open actions in this issue are now redundant. What do you think?

Thanks Adrian

**Comment by AdrianHammond** [comment]
Hi @mcleo-d  - should we close off this issue as its now been superseded by #253 ?

Thanks Adrian

**Comment by mcleo-d** [comment]
Development has now moved into https://github.com/finos/compliant-financial-infrastructure/issues/253

Thank you @AdrianHammond and Red Hat for the hard work on this issue 👍🏻
